### DISCUSSION TITLE
ImageMagick Latest Adoption

### DISCUSSION THREAD

**Discussion by programmerarunkumar** [ImageMagick/ImageMagick#7174] [Help]
Currently Im having ImageMagick 7.1.0-60

generally whats the recommended interval to adopt the latest imagemagick version ?

How i can find the major version from the [imagemagick release list](https://github.com/ImageMagick/ImageMagick/releases) ?

**Comment by programmerarunkumar** [comment]
Can someone clarify this ?

**Reply by dlemstra** [reply]
Can you please be a bit more patient? This is something we do in our spare time and no-one has had time to respond yet.

**Reply by programmerarunkumar** [reply]
Okay. sorry for the inconvenience

**Comment by snibgo** [comment]
> generally whats the recommended interval to adopt the latest imagemagick version ?

I don't think there is a general answer. It depends on your usage.

If you develop systems and new features in IM are useful to you, then you need to use fairly current versions. On the other hand, if you manage a mature system then stability is more important and upgrading (including regression testing) is expensive, so you might upgrade only when you are forced to.

**Reply by programmerarunkumar** [reply]
Yes. I can't afford upgrading frequently

suppose just for an assumption if there is any security issue or any major issue in my current ImageMagick Version. how can i get to know about this ?
so i can update to latest version
